operator/v1/types_network.go

```diff
@@ -440,6 +440,10 @@ type OVNKubernetesConfig struct {
 	// cluster.
 	// +optional
 	IPsecConfig *IPsecConfig `json:"ipsecConfig,omitempty"`
+	// ipsecExtern enables external (NS) IPsec for the node, the configuration
+	// is done via nmstate operator.
+	// +optional
+	IPsecExtern *IPsecExtern `json:"ipsecExtern,omitempty"`
```

Comment:
since this is customer facing
I think we should use the full word and not a shortcut
ie - IPsecExternal

you are not saving much by removing 2 letters anyhow

Reply:
Done